ChartboostMediationAdapterBidMachine.podspec

```diff
@@ -23,7 +23,7 @@ Pod::Spec.new do |spec|
   spec.dependency 'ChartboostMediationSDK', '~> 4.0'
 
   # Partner network SDK and version that this adapter is certified to work with.
-  spec.dependency 'BidMachine', '2.3.0'
+  spec.dependency 'BidMachine', '~> 2.4.0.0'
```

Comment:
@kendallrogers BidMachine uses four digit version numbers, but I'm not sure this is the right thing to do in the adapter podfile. Should I remove the last digit so the floating peg works for 2.4.x ?